### ISSUE TITLE
Unable to access meatball menu while selecting an account to autofill

### ISSUE THREAD

**Issue by candroid-man** [bitwarden/mobile#2482]
### Steps To Reproduce

1. Go to a login page
2. Click on the Bitwarden autofill button
3. Click on the meatball menu (aka 3 horizontal dot menu)

### Expected Result

You should be able to access this menu to view and edit accounts.

### Actual Result

You are simply unable to click on the icon, it won't do anything

### Operating System Version

Android 13

### Device

Google Pixel 6 Pro

### Build Version

2023.3.2 (6098)

**Comment by rtmor** [comment]
Experiencing the same.

```
Operating System Version: Android 13
Device: Samsung Galaxy S21
Build Version: 2023.3.2 (6098)
```

**Comment by c1-jack** [comment]
Hi there!

Thank you for your report, it seems like it is a duplicate of this one: https://github.com/bitwarden/mobile/issues/2446

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time in there - our engineering team will be happy to review these.

This issue will now be closed.

Thanks!

**Comment by typohandwerker** [comment]
Why was this closed? Error is still in actual version 2023.3.2, Android 13, Pixel 5

**Comment by cksapp** [comment]
> Why was this closed? Error is still in actual version 2023.3.2, Android 13, Pixel 5

As described above this was closed as it is a duplicate of #2446, which was subsequently closed in response to a PR that addressed this.

That PR _(#2460)_ has been merged into the main codebase recently and should be fixed in an upcoming release.
Most likely `2023.4.0` or something similar
